Regardless of the discussion above, the package also doesn't build because of the cmake 4 update and CI cannot verify if the package now builds.

```
unstable-2019-03-19 -DENABLE_SHARED=ON
copying path '/nix/store/fkc92m9j19swk3dw8yxpgdql26nd6rb3-glslang-16.0.0-dev' from 'https://cache.nixos.org'...
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.
  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.
  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
-- Configuring incomplete, errors occurred!
error: builder for '/nix/store/6avdkxkd6i6mij8y0ls68rpfc8wpbvqb-ffts-0-unstable-2019-03-19.drv' failed with exit code 1;
```

There should be a pinned issue regarding that.

--- a/pkgs/by-name/sc/scopehal-apps/package.nix
+++ b/pkgs/by-name/sc/scopehal-apps/package.nix
@@ -88,6 +88,11 @@ stdenv.mkDerivation {
     "-DNGSCOPECLIENT_VERSION=${version}"
   ];
 
+  env.NIX_CFLAGS_COMPILE = toString [
+    # error: variable 'empty_string' is uninitialized when passed as a const pointer argument here [-Werror,-Wuninitialized-const-pointer]
+    "-Wno-error=uninitialized"
+  ];
+
   patches = [
     ./remove-git-derived-version.patch
   ]

